subsys/net/lib/openthread/platform/openthread-core-zephyr-config.h

```diff
@@ -142,4 +142,30 @@
  */
 #define RADIO_CONFIG_SRC_MATCH_EXT_ENTRY_NUM 0
 
+/**
+ * @def OPENTHREAD_CONFIG_PLAT_LOG_FUNCTION
+ *
+ * The platform logging function for openthread.
+ *
+ */
+
+#define _OT_CONF_PLAT_LOG_FUN_NARGS__IMPL(		\
+		_0, _1, _2, _3, _4, _5, _6, _7, _8, _9, _10,\
+		_11, _12, _13, _14, N, ...) N
+
+#define _OT_CONF_PLAT_LOG_FUN_NARGS__GET(...) \
```

Comment:
check `util.h` where `NUM_VA_ARGS_LESS_1` is present if that is not enough then it would be good to extend `util.h`

Reply:
I agree with Tomasz, `util.h` contain a lot of macros with quite generic names (MIN, MAX etc), I'd rather avoid including it in OT. Who knows what issues we could run into (if not today, perhaps in the future).